src/spotify/senum.rs

```diff
@@ -1219,5 +1219,6 @@ pub enum DeviceType {
     Computer,
     Smartphone,
     Speaker,
+    TV,
```

Comment:
Perhaps You should update the related Doc about Device type in case of confusing other developers.

Reply:
Good point. I just added the remaining device types and put a link in the docs.

Reply:
Cool